### PR TITLE
pdpb: synchronizing region should support bucket.

### PR DESCRIPTION
**Pull request by bufferflies**
Signed-off-by: bufferflies <1045931706@qq.com>
synchronizing region should support bucket.

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -448,6 +448,9 @@ message RegionHeartbeatResponse {
     //       1. replacing peers
     //       2. demoting voter directly
     ChangePeerV2 change_peer_v2 = 9;
+
+    // PD should contain the buckets information if origin region's epoch is changed.
+    metapb.Buckets buckets=10;
```
Comment (BusyJay) [inline proto/pdpb.proto:453]: What's the purpose? And it should be formatted.

**Comment by disksing** [comment]
@bufferflies any update?

File: proto/pdpb.proto (hunk 1 above)
Comment (bufferflies) [reply proto/pdpb.proto:453]: the operator should have buckets information like transterLeader.

**Comment by bufferflies** [comment]
PTAL @BusyJay again

**Review by BusyJay — APPROVED** [review]

**Pull request merged**